CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 ## next
 
+## 2.1.4
+
 *Other*
```

Comment:
Should we just drop the other header? If not, 781 & 765 look more like `Enhancements` than other to me.